svm/tests/example-programs/write-to-account/Cargo.toml

```diff
@@ -4,7 +4,6 @@ version = "2.3.0"
 edition = "2021"
 
 [dependencies]
-solana-program = { path = "../../../../sdk/program", version = "=2.2.0" }
```

Comment:
nit: Since we're removing the only dep, should we remove the above `[dependencies]` line too ?

Reply:
Nevermind - I see other lines of code that we're actually compiling that now have empty `dependency` list so this is fine as is. Leaving the original comment + this response incase anyone else was wondering same thing